keyboards/rgbkb/zygomorph/rev1/config.h

```diff
@@ -55,7 +55,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     #define RGBLED_NUM 60
     #define RGBLED_SPLIT { 30, 30 }
 #endif
-#define DRIVER_LED_TOTAL 30
+
+#define DRIVER_LED_TOTAL 60
+#define RGB_MATRIX_SPLIT { 30, 30 }
```

Comment:
```suggestion
#define RGB_MATRIX_SPLIT { 30, 30 }
#define SPLIT_TRANSPORT_MIRROR
```

Reply:
The split rgb matrix work is only enabled for my keymap, as the base zygomorph build instructions don't support split rgb matrix as you have to desolder a jumper.